tests/decimal_tests.rs

```diff
@@ -4572,6 +4572,19 @@ mod generated {
     gen_test!(test_sub_111_111, "Sub_111_111.csv", checked_sub);
 }
 
+#[cfg(feature = "proptest")]
```

Comment:
Thanks for adding a test! Could you add it to the build too? Probably inside the `misc.toml` file: https://github.com/paupino/rust-decimal/blob/master/make/tests/misc.toml

Reply:
Added. Thanks for pointing it out.